cli/src/main/kotlin/StartCommand.kt

```diff
@@ -45,41 +45,34 @@ class StartCommand(private val config: OrtServerOptions) : SuspendingCliktComman
         help = "The ID of the repository."
     ).long().required()
 
-    private val revision by option(
-        "--vcs-revision",
-        envvar = "VCS_REVISION",
-        help = "Revision of the repository to start the run for."
-    ).required()
-
-    private val path by option(
-        "--vcs-sub-path",
-        envvar = "VCS_SUB_PATH",
-        help = "Optional VCS sub-path of the repository."
-    )
-
-    private val jobConfigContext by option(
-        "--job-config-context",
-        envvar = "JOB_CONFIG_CONTEXT",
-        help = "Configuration context to use for the run."
-    )
-
     private val wait by option(
         "--wait",
         envvar = "WAIT",
         help = "Wait for the run to finish."
     ).flag()
 
+    private val ortRunConfigFile by option(
+        "--ort-run-config-file",
+        envvar = "ORT_RUN_CONFIG_FILE",
+        help = "The path to the Create ORT run configuration file."
+    ).file()
+
+    private val ortRunConfigString by option(
+        "--ort-run-config-string",
```

Comment:
We should drop the `string` part. CLI parameters are by definition a `string`.
I don't think that you need this to distinguish from the `file` version of the option